pkg/daemon/update_test.go

```diff
@@ -274,6 +274,16 @@ func TestKernelAguments(t *testing.T) {
 			deletions: []string{"nosmt", "baz=test"},
 			additions: []string{"hello=world"},
 		},
+		{
+			args:      []string{"foo", "bar=1 hello=world"},
+			deletions: []string{"nosmt", "baz=test"},
+			additions: []string{"bar=1", "hello=world"},
+		},
+		{
+			args:      []string{"foo", " baz=test bar=\"hello world\""},
```

Comment:
Not sure if space after opening quote `" baz=test` is intentional

Reply:
Yeah, it was intentional. Wanted to test that leading spaces get trimmed appropriately.